snap/snapcraft.yaml

```diff
@@ -21,130 +21,130 @@ plugs:
     bus: system
     name: io.openthread.BorderRouter.wpan0
 
+  # Uses system-files snap interface to access the gpiochip0
+  dev-gpiochip0:
+    interface: system-files
+    read:
+      - /dev/gpiochip0
+      - /sys/devices/platform/soc/fe200000.gpio/gpiochip0/dev
+    write: 
+      - /dev/gpiochip0
```

Comment:
`write` implies both write and read; see https://snapcraft.io/docs/system-files-interface

Reply:
So it's not necessary to repeat the `/dev/gpiochip0`  on both, good to know, thank you.